### ISSUE TITLE
ShareX suddenly wiped my settings

### ISSUE THREAD

**Issue by d3athrow** [ShareX/ShareX#4230]
Yesterday my sharex settings were perfectly fine, today on boot up everything is reset to default. I tried first by manually restoring backups of the json files but sharex seemingly ignores them. Afterwards I tried importing an old backup file but i get pic related
![this](https://i.imgur.com/3dtWzof.png)

The application path is set correctly and the json files there have the old settings in them but it's just being ignored in favor of whatever the internal defaults are.

![path](https://i.imgur.com/wETRSkU.png)

And I guess [here](https://www.pastiebin.com/5d31d54bd05f8) is relevant logs showing an error loading the settings.

**Comment by dangeles84** [comment]
This also seems to happen when ShareX forces a restart, eg, when you update the path of your personal folder and ShareX asks to restart. Doing so resets all your hotkeys, paths, workflows, etc.

**Comment by Jaex** [comment]
@dangeles84 If personal folder you selected already had settings, then it would overwrite those settings otherwise. So its working as intended.

**Comment by dangeles84** [comment]
@Jaex Ah, okay.  I should have clarified. Firstly, I experienced the above issue where my settings reverted to the default.  I then made some changes to my configuration, then realized that my screenshots were going to a different folder.  After pointing ShareX to the location I originally used, the settings were once again wiped to the default (with the exception of the path).  So, I too experienced a reset of my settings (which I did not initiate), but the settings resetting after updating my screenshot folder is WAD.  Thanks!
